.nuxt/App.vue

```diff
@@ -0,0 +1,88 @@
+<template>
```

Comment:
We don't commit .nuxt/** folder. Or more generally, we don't commit auto-generated code until some weird scenario pops up.